srtcore/core.cpp

```diff
@@ -7712,8 +7766,10 @@ int CUDT::packLostData(CPacket& w_packet, steady_clock::time_point& w_origintime
             int32_t seqpair[2];
             seqpair[0] = w_packet.m_iSeqNo;
             seqpair[1] = CSeqNo::incseq(seqpair[0], msglen);
-            sendCtrl(UMSG_DROPREQ, &w_packet.m_iMsgNo, seqpair, 8);
 
+#ifndef SRT_TEST_DISABLE_KEY_CONTROL_PACKETS
+            sendCtrl(UMSG_DROPREQ, &w_packet.m_iMsgNo, seqpair, sizeof(seqpair));
+#endif
```

Comment:
I don't think we should add this to the master. This will be forgotten eventually.
I would suggest to create a document, and maybe a patch, that describe how to perform this test.
Roughlt speaking. disable all `sendCtrl(UMSG_DROPREQ...`

Reply:
This `sendCtrl` must remain here - that's the intention of the fix. The macrodefinition is here so that tests can be made that simulate dropping this packet.

Reply:
I was talking about the ifdef statement. **Code pollution**. 
SRT_TEST_DISABLE_KEY_CONTROL_PACKETS can be easily added inside of the `sendCtrl()` when needed. But they should not exist in master.

Reply:
Ah ok, then you are right. Moreover, with more advanced testing we can disable them using some better methods.